tests/commands/update/test_update_case_cmd.py

```diff
@@ -1,141 +1,214 @@
 # -*- coding: utf-8 -*-
-from scout.demo import clinical_snv_path, clinical_sv_path
 
 from scout.commands import cli
 from scout.server.extensions import store
 
-def test_update_case(mock_app, case_obj):
+def test_update_case_no_args(mock_app):
     """Tests the CLI that updates a case"""
 
+    ## GIVEN a CLI object
     runner = mock_app.test_cli_runner()
-    assert runner
 
-    # Test CLI base, no arguments provided
+    ## WHEN updating a case without any args
     result =  runner.invoke(cli, ['update', 'case'])
 
-    # it should return error message
+    ## THEN it should return an error message
     assert 'Please specify which case to update' in result.output
 
-    # provide a case id
-    result =  runner.invoke(cli, ['update', 'case',
-        case_obj['_id'],
-        ])
-    assert 'INFO Fetching case {}'.format(case_obj['_id']) in result.output
+def test_update_case_wrong_collaborator(mock_app, case_obj):
+    """Tests the CLI that updates a case"""
+
+    ## GIVEN a CLI object
+    runner = mock_app.test_cli_runner()
 
-    # provide a case id and a collaborator whis is not valid
+    ## WHEN providing a case id and a collaborator whis is NOT valid
     result =  runner.invoke(cli, ['update', 'case',
         case_obj['_id'],
         '-c', 'cust666'
         ])
+    
+    ## THEN assert that there is a warning in the output
     assert 'Institute cust666 could not be found' in result.output
 
-    # provide a case id and a valid collaborator
+def test_update_case_existing_collaborator(mock_app, case_obj):
+    """Tests the CLI that updates a case"""
+
+    ## GIVEN a CLI object
+    runner = mock_app.test_cli_runner()
+
+    ## WHEN providing a case id and a valid collaborator
     result =  runner.invoke(cli, ['update', 'case',
         case_obj['_id'],
         '-c', 'cust000'
         ])
+    
+    ## THEN assert that the case is fetched
     assert 'INFO Fetching case {}'.format(case_obj['_id']) in result.output
 
-    # try first to remove collaborator from case object and to add it via the CLI
+def test_update_case_change_collaborator(mock_app, case_obj):
+    """Tests the CLI that updates a case"""
+
+    ## GIVEN a CLI object
+    runner = mock_app.test_cli_runner()
+
+    ## WHEN removing collaborator from case object
     store.case_collection.find_one_and_update(
         {'_id': case_obj['_id']},
         {'$set' : {'collaborators' : [] }}
     )
-    assert store.case_collection.find({
-        '_id': case_obj['_id'], 'collaborators' : []
-    }).count() == 1
-    # provide a case id and a valid collaborator and see that collaborator is added to case
+    res = store.case_collection.find_one({'_id': case_obj['_id']})
+    assert res['collaborators'] == []
+
+    ## WHEN a collaborator is added via the CLI
     result =  runner.invoke(cli, ['update', 'case',
         case_obj['_id'],
         '-c', 'cust000'
         ])
-    assert 'INFO Fetching case {}'.format(case_obj['_id']) in result.output
```

Comment:
We don't care about what the CLI prints ?

Reply:
Not so important but I like that you test it! Did I remove that or just changed it?